uwblueprint/sistering
# Employee Model Migration

## Ticket link

Closes #442


## Implementation description
* Employee has a list of branches associated with it instead of a singular branch
* Branches are now implemented the same as the volunteer users


## Steps to test
1. Run the following query 
query AdminUserManagementPage_Users {
  employeeUsers {
    branches {
      id
      name
    }
  }
}
verify the following is returned
{
  "data": {
    "employeeUsers": [
      {
        "branches": [
          {
            "id": "1",
            "name": "Kitchen"
          }
        ]
      },
      {
        "branches": [
          {
            "id": "2",
            "name": "Arts"
          }
        ]
      }
    ]
  }
}



## What should reviewers focus on?
* userService.ts functions that deal with employeeUsers should stay the same


## Checklist
